docs/book/src/developer/tilt.md

```diff
@@ -37,10 +37,35 @@ make envsubst
 
 First, make sure you have a kind cluster and that your `KUBECONFIG` is set up correctly:
 
-``` bash
+{{#tabs name:"install-kind" tabs:">=v0.9.x, Docker infrastructure provider - CAPD"}}
+{{#tab >=v0.9.x}}
+
+```bash
 kind create cluster
 ```
 
+{{#/tab }}
+{{#tab Docker infrastructure provider - CAPD}}
+
+Run the following command to create a kind config file for allowing the Docker provider to access Docker on the host:
+
+```bash
+cat > kind-cluster-with-extramounts.yaml <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+    - hostPath: /var/run/docker.sock
+      containerPath: /var/run/docker.sock
+EOF
+```
+
+Then follow the instruction for your kind version using  `kind create cluster --config kind-cluster-with-extramounts.yaml`
+to create the management cluster using the above file.
```

Comment:
I agree we should give those instructions to people using CAPD, but given that we already have those instructions in the quick start, so I'm -1 to duplicate them.
What about adding a note linking to the existing instructions, something like "IMPORTANT, if you are planning to use the CAPD provider, you have create required mounts for allowing the provider to access the Docker socket on the host; see [quick start]() for instructions"